src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -1090,6 +1090,14 @@
     <value>Pane animations</value>
     <comment>Header for a control to toggle animations on panes. "Enabled" value enables the animations.</comment>
   </data>
+  <data name="Globals_AlwaysShowTrayIcon.Header" xml:space="preserve">
+    <value>Always show the Tray Icon</value>

```

Comment:
We may want to align with Windows official words here. "Always display an icon in the notification area"